SwiftUIDZ/SwiftUIDZ/ProfileInfo.swift

```diff
@@ -0,0 +1,62 @@
+// ProfileInfo.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Профиль
+struct ProfileInfo {
+    let imageName: String
+    let name: String
+    let badge: Bool
+    let badgeNumber: String?
+    let destination: AnyView?
```

Comment:
```suggestion
    let imageName: String
    let name: String
    let badge: Bool
    let badgeNumber: String?
    let destination: AnyView?
```

тут описание не просят?

Reply:
просят